api/server/server.go

```diff
@@ -895,6 +887,11 @@ func deleteContainers(eng *engine.Engine, version version.Version, w http.Respon
 	}
 
 	if err := d.ContainerRm(name, config); err != nil {
+		errStr := strings.ToLower(err.Error())
+		// Force a 404 for the empty string
+		if strings.Contains(errStr, "prefix can't be empty") {
```

Comment:
since you don't need the `errStr` var anywhere else, you can put the `strings.ToLower..` line as the first parameter to `strings.Contains()`